src/main/java/pl/lodz/p/it/zzpj/botsite/services/UserTaskService.java

```diff
@@ -0,0 +1,22 @@
+package pl.lodz.p.it.zzpj.botsite.services;
+
+import pl.lodz.p.it.zzpj.botsite.entities.UserTask;
+import pl.lodz.p.it.zzpj.botsite.exceptions.UserTaskIdAlreadyExistsException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.UserTaskNotFoundException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.UserTaskRetrievalException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.UserTaskStatusException;
+
+import java.time.DateTimeException;
+import java.util.Date;
+import java.util.List;
+
+public interface UserTaskService {
+    void addUserTask(UserTask task) throws UserTaskIdAlreadyExistsException;
+    UserTask findById(String id) throws UserTaskRetrievalException;
+    List<UserTask> findListByUserId(String userId) throws UserTaskNotFoundException;
```

Comment:
findList? Nazwa nic nie sugeruje honestly. Posiadając obiekt trzeba się domyślać co by takie cuś robiło :)

Reply:
Co do tych nazw to one nie są w ogóle zaimplementowane i aktualnie zastanawiam się czy są potrzebne i co maja dokładnie zwracać 😝 Chciałem to przedyskutować.